README.md

```diff
@@ -0,0 +1,37 @@
+# GOB-Airflow
+
+GOB Airflow orchestrates GOB data pipelines.
+
+# Infrastructure
+
+A running [GOB infrastructure](https://github.com/Amsterdam/GOB-Infra)
+is required to run this component.
+
+You might want to stop GOB-Workflow however to avoid conflicts with queue's handling.
+
```

Comment:
This conflicts with the requirement that GOB workflows should be possible with different workflow managers. One is GOB Workflow another is Airflow and these should run besides each other. If this is not possible in this solution then explain what the reason is.